src/pages/Detail/Detail.js

```diff
@@ -0,0 +1,34 @@
+import React, { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import styled from 'styled-components';
+import DetailAside from './DetailAside/DetailAside';
+import DetailContent from './DetailContent/DetailContent';
+import { API } from '../../config';
+
+const Detail = () => {
+  const [data, setData] = useState({});
```

Comment:
```suggestion
  const [hotelData, setHotelData] = useState({});
```
* 변수명 만 봐도 어떤 데이터 일지 유추할 수 있게 지어주시는게 좋겠습니다!

Reply:
전반적으로 해당 state를 `hotelData`로 수정하였고, 하위 컴포넌트에 내려주던 props 이름 역시 `data`에서 `hotelData`로 수정하였습니다.